### PR TITLE
fix: update anon info

### PR DESCRIPTION
**Pull request by brm-stnd**
### Summary by CodeRabbit

```
- Bug Fix: Corrected the population of anonymous user details in chat initialization. This fix ensures that the chat feature works properly for anonymous users, enhancing their experience and interaction within the platform.
```

### CODE REVIEW

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 195c24f977293a60d8bfab6e88c43a00977ad5fc and 1b95e9ad4f38c2e294ec984e0d5b838973d268f6 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* controllers/chat/initChatFromPost.js (1)
</details>






---

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 195c24f977293a60d8bfab6e88c43a00977ad5fc and 1b95e9ad4f38c2e294ec984e0d5b838973d268f6 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* controllers/chat/initChatFromPost.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* controllers/chat/initChatFromPost.js

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

File: controllers/chat/initChatFromPost.js [hunk 1]
```diff
@@ -114,21 +114,18 @@ const initChatFromPost = async (req, res) => {
...
         if (userModel.is_anonymous) {
           channelType = CHANNEL_TYPE.ANONYMOUS;
-        }
-
-        let anonDetailInfo = {};
-        if (targetUserModel.is_anonymous) {
           anonDetailInfo = {
             anon_user_info_color_code:
-              newStateMemberWithAnonInfo[targetUserModel.user_id].anon_user_info_color_code || null,
+              newStateMemberWithAnonInfo[userModel.user_id].anon_user_info_color_code,
```
Comment (n-permana) [inline controllers/chat/initChatFromPost.js:122]: mungkin bisa dibuat jadi 
const memberWithAnonInfo = newStateMemberWithAnonInfo[userModel.user_id]
memberWithAnonInfo.anon_user_info_color_code
memberWithAnonInfo.anon_user_info_color_name
memberWithAnonInfo.anon_user_info_emoji_code
memberWithAnonInfo.anon_user_info_emoji_name
jadi bisa dilakukan pengecekan dulu apa datanya ada atau engga.

**Review by n-permana — APPROVED** [review]

File: controllers/chat/initChatFromPost.js (hunk 1 above)
Comment (brm-stnd) [reply controllers/chat/initChatFromPost.js:122]: ini knp saya hilangkan ppengecekan datanya karena di bagian atas line2 sebelumnya sudah saya define mas bisa datanya tidak ada, jadi dibawah sini jika user anon pasti sudah terdefine mas datanya

**Pull request merged**